src/defaults/html.pipe.js

```diff
@@ -43,8 +44,11 @@ const htmlpipe = (cont, payload, action) => {
     .before(fetch)
     .when(({ content }) => !(content !== undefined && content.body !== undefined))
     .before(parse)
+    .before(parse_frontmatter)
+    .before(({ content }) => console.log("AFT PARSE_FM", content.mdast))
```

Comment:
In development mode, we dump the entire pipeline content into the `logs/debug` directory.

Reply:
Jaja, das ist auch nur für mich zum debuggen…